pkg/pillar/types/zfs.go

```diff
@@ -99,10 +169,37 @@ type DiskDescription struct {
 	Serial      string // serial number of disk
 }
 
+// ZIOType - IO types in ZFS
+const (
+	ZIOTypeNull = iota
+	ZIOTypeRead
+	ZIOTypeWrite
+	ZIOTypeFree
+	ZIOTypeClaim
+	ZIOTypeIOCtl
+	ZIOTypes
```

Comment:
What is this last value about?

Reply:
This value determines the number of ZIOType enums (currently 6). (Should always be the last)
I use it to initialize an array with metrics and as a limiter (i < ZIOTypes) in a loop when collecting data.

I added a comment for this value in #2645 (First part)

Reply:
Corrected